src/utils/sidecar.js

```diff
@@ -4,8 +4,21 @@ export const openSidecar = async (program, type, data) => {
     const sidecar = Command.sidecar(program);
 
     const promiseDataReceived = new Promise((resolve, reject) => {
-        sidecar.stdout.on('data', e => resolve(e));
-        sidecar.stderr.on('data', e => reject(e));
+        // Listen for output
+        const stdout = [];
+        sidecar.stdout.on('data', e => stdout.push(e));
+        let hadStderr = false;
+        sidecar.stderr.on('data', e => {
+            if (!hadStderr)
+                reject(e);
+            hadStderr = true;
+        });
```

Comment:
I don't think the flag is neeeded, just return after rejecting.

Reply:
Should be resolved with latest commit